outputs.tf

```diff
@@ -48,9 +43,9 @@ output "public_subnets" {
   description = "The public subnets in the VPC."
 }
 
-output "read_terraform_state_role" {
-  value       = aws_iam_role.read_terraform_state
-  description = "The role that allows read-only access to the cool-sharedservices-networking state in the Terraform state bucket."
+output "read_terraform_state" {
+  value       = module.read_terraform_state
+  description = "The IAM policies and role that allow read-only access to the cool-sharedservices-networking state in the Terraform state bucket."
```

Comment:
It never occurred to me to output an entire module.  C'est intéressant.

Reply:
I didn't see any downsides to doing it this way, so I figured I'd give it a shot.